code/game/objects/items/robot/robot_items.dm

```diff
@@ -866,7 +866,6 @@
 	. = ..()
 	if(wrapped)
 		var/mutable_appearance/wrapped_appearance = mutable_appearance(wrapped.icon, wrapped.icon_state)
-		wrapped_appearance.overlays = wrapped.overlays.Copy()
```

Comment:
this probably shouldn't be reverted since it's unrelated to the functionality of the gripper